csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -562,6 +561,39 @@ List of URLs (`urls`) of acknowledgment is a container (value type `array`) for
 Any URL of acknowledgment contains the URL or location of the reference to be acknowledged.
 Value type is string with format URI (`uri`).
 
+##### 3.1.1.5 Acknowledgments Type - Example
+
+```
+"acknowledgments": [
+  {
+    "names": [
+      "Johann Sebastian Bach",
+      "Georg Philipp Telemann",
+      "Georg Friedrich Händel"
+    ],
+    "organization": "Baroque composers",
+    "summary": "wonderful music"
+  },
+  {
+    "organization": "CISA"
+    "summary": "coordination efforts",
+    "urls": [
+      "https://cisa.gov"
+    ]
+  },
+  {
+    "organization": "BSI",
+    "summary": "assistance in coordination"
+  },
+  {
+    "names": [
+      "Antonio Vivaldi"
+    ],
+    "summary": "influencing other composers"
+  },
+]
+```
+
```

Comment:
```suggestion

The example above should lead to the following outcome in a human-readable advisory:

This should lead in a human-readable advisory to the following outcome:

> We thank the following parties for their efforts:
> 
> * Johann Sebastian Bach, Georg Philipp Telemann, Georg Friedrich Händel from Baroque composers for wonderful music
> * CISA for coordination efforts (see: https://cisa.gov)
> * BSI for assistance in coordination
> * Antonio Vivaldi for influencing other composers

```

Reply:
Can you please fix the double leading clause here so I can just accept this suggestion? :-)

Reply:
Sorry that was by accident.